ui/components/app/fee-details-component/fee-details-component.js

```diff
@@ -29,9 +29,8 @@ import { useGasFeeContext } from '../../../contexts/gasFee';
 
 export default function FeeDetailsComponent({
   txData,
-  supportsEIP1559,
   useCurrencyRateCheck,
-  shouldShow = true,
+  hideGasDetails = false,
```

Comment:
is there a reason we are renaming this to `hideGasDetails` instead of `shouldShow` or `showFeeDetails`?